qemu/qemu.go

```diff
@@ -1502,13 +1502,13 @@ func (config *Config) appendMemoryKnobs() {
 		if config.Memory.Size != "" {
 			dimmName := "dimm1"
 			objMemParam := "memory-backend-ram,id=" + dimmName + ",size=" + config.Memory.Size + ",prealloc=on"
-			deviceMemParam := "pc-dimm,id=" + dimmName + ",memdev=" + dimmName
+			numaMemParam := "node,memdev=" + dimmName
```

Comment:
Do we expect NUMA emulation to be enabled in this use case?

Reply:
For all the other cases in `appendMemoryKnobs`, numa node is added as well.